docs/tutorial/tricks.md

```diff
@@ -0,0 +1,141 @@
+# Common tricks used in (T)FHE
+
+As explained in the [Basics of FHE](../getting-started/fhe_basics.md), the challenge for a developer
```

Comment:
... the challenge for developers is to adapt their code to fit FHE ...

Reply:
fixed